test/integration/tcp_proxy_integration_test.cc

```diff
@@ -0,0 +1,125 @@
+#include "test/integration/tcp_proxy_integration_test.h"
+
+#include "common/network/utility.h"
+#include "common/ssl/context_manager_impl.h"
+
+#include "test/integration/ssl_utility.h"
+#include "test/integration/utility.h"
+#include "test/mocks/runtime/mocks.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
```

Comment:
I'm not suggesting we put it in the anonymous namespace at outer scope. I'm suggesting we add an additional anonymous namespace within the existing set of namespace scopes at inner scope. This doesn't imply additional verbosity, and is safer when we link into the single coverage binary.